README.md

```diff
@@ -31,14 +31,22 @@ Just clone the repo by `git clone https://github.com/JetBrains-Research/ast-tran
 
 Run the command `./gradlew :cli -Pinput=<Input directory with python files> -Poutput=<Output directory> -Pyaml=<YAML config path>`.
 The YAML configuration file example can be found [here](./config.yaml).
-Set of the transformations is applied `n transformations` times. Each round for each transformation the random  
-number between `0.0` and `1.0` is generated. If this value less than `p` for the current transformation,
-the transformation will be applied. The result for each round is stored in the separated folder.
+Set of the transformations is applied `n apply` times. Each round for each transformation the random  
+number between `0.0` and `1.0` is generated. If this value is less than `p` for the current transformation,
+the transformation will be applied to the source files. The result for each round is stored in the separated folder.
+
+For example, the source files are stored in the `source` folder, `n apply` is `5`, and we have list 
+of the `3` transformations. `5` times will be executing the following steps:
```

Comment:
we have list of the `3` transformations -> we have a list of `3` transformations